packages/libzip/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://libzip.org/
+TERMUX_PKG_DESCRIPTION="library for reading, creating, and modifying zip archives."
```

Comment:
No period at end: it's a brief description, not a sentence.

Reply:
https://github.com/termux/termux-packages/pull/3433/commits/1a5b7fbaa7517efec2696bc9a16db0daeb4c2b13